features/google/test/mocks/i18n-mock.js

```diff
@@ -0,0 +1,5 @@
+export default {
```

Comment:
Unrelated?

Reply:
It might not with these specific tests, but remember that as soon as any file that includes !silly-i18n is pulled in, this needs to be included to generate the tested files.